packages/main/src/themes/sap_horizon_dark/RadioButton-parameters.css

```diff
@@ -1,9 +1,16 @@
 @import "../base/RadioButton-parameters.css";
 
 :root {
-	--_ui5_radio_button_border_width: 0.125rem;
-	--_ui5_radio_button_border_radius: 0.75rem;
+	--_ui5_radio_button_border_width: var(--sapContent_FocusWidth);
+	--_ui5_radio_button_border_radius: 0.5rem;
 	--_ui5_radio_button_label_color: var(--sapField_TextColor);
-	--_ui5_radio_button_border_readonly_focus_style: dashed;
-	--_ui5_radio_button_focus_dist: 0.375rem;
+	--_ui5_radio_button_inner_ring_radius: 27.5%;
+	/* --_ui5_radio_button_outer_ring_hover_shadow: 0 0 0.125rem fade(var(--sapNeutralColor), 72);  does not work??? */
```

Comment:
This doesn't work in regular CSS because `fade` is a function that [comes with LESS](https://lesscss.org/functions/#color-operations-fade).